test/lib/validation/schema/specVersion/kind/project/application.js

```diff
@@ -46,7 +46,253 @@ test.after.always((t) => {
 	t.context.ajvCoverage.verify(thresholds);
 });
 
-SpecificationVersion.getVersionsForRange(">=2.0").forEach(function(specVersion) {
+SpecificationVersion.getVersionsForRange(">=4.0").forEach(function(specVersion) {
+	test(`Valid configuration (specVersion ${specVersion})`, async (t) => {
+		await assertValidation(t, {
+			"specVersion": specVersion,
+			"kind": "project",
+			"type": "application",
+			"metadata": {
+				"name": "com.sap.ui5.test",
+				"copyright": "okay"
+			},
+			"resources": {
+				"configuration": {
+					"propertiesFileSourceEncoding": "UTF-8",
+					"paths": {
+						"webapp": "/my/path"
+					}
+				}
+			},
+			"builder": {
+				"resources": {
+					"excludes": [
+						"/resources/some/project/name/test_results/**",
+						"/test-resources/**",
+						"!/test-resources/some/project/name/demo-app/**"
+					]
+				},
+				"bundles": [
+					{
+						"bundleDefinition": {
+							"name": "sap-ui-custom.js",
+							"defaultFileTypes": [
+								".js"
+							],
+							"sections": [
+								{
+									"name": "my-raw-section",
+									"mode": "raw",
+									"filters": [
+										"ui5loader-autoconfig.js"
+									],
+									"resolve": true,
+									"resolveConditional": true,
+									"renderer": true,
+									"sort": true
+								},
+								{
+									"mode": "provided",
+									"filters": [
+										"ui5loader-autoconfig.js"
+									],
+									"resolve": false,
+									"resolveConditional": false,
+									"renderer": false,
+									"sort": false,
+									"declareRawModules": true
+								}
+							]
+						},
+						"bundleOptions": {
+							"optimize": true,
+							"decorateBootstrapModule": true,
+							"addTryCatchRestartWrapper": true
+						}
+					},
+					{
+						"bundleDefinition": {
+							"name": "app.js",
+							"defaultFileTypes": [
+								".js"
+							],
+							"sections": [
+								{
+									"name": "some-app-preload",
+									"mode": "preload",
+									"filters": [
+										"some/app/Component.js"
+									],
+									"resolve": true,
+									"sort": true,
+									"declareRawModules": false
+								},
+								{
+									"mode": "require",
+									"filters": [
+										"ui5loader-autoconfig.js"
+									],
+									"resolve": true
```

Comment:
```suggestion
									"resolve": true,
									"async": false
```

Test the new property